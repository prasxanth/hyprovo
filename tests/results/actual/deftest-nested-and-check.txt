2021-11-24 14:41:21.337 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results): add pass
2021-11-24 14:41:21.339 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results): sub fail
2021-11-24 14:41:21.344 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 14:41:21.346 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results / test-sub): sub fail
2021-11-24 14:41:21.351 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 14:41:21.353 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results / test-div): div fail
2021-11-24 14:41:21.353 - DEBUG - (test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
