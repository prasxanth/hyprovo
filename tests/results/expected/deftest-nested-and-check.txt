2021-11-24 14:10:30.821 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results): add pass
2021-11-24 14:10:30.823 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results): sub fail
2021-11-24 14:10:30.827 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 14:10:30.829 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results / test-sub): sub fail
2021-11-24 14:10:30.833 - INFO - Pass ✓... (test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 14:10:30.835 - ERROR - Fail ✗... (test-arithmetic-1-report-combined-results / test-div): div fail
2021-11-24 14:10:30.835 - DEBUG - (test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
